### PR TITLE
[G&M] Expose extraction log messages

### PR DESCRIPTION
**Pull request by nyrb7**
This change creates a new status icon button that shows a color representation of the status of the most recent extraction job. It also creates a modal that opens when one of the extraction logs contains issues. 

https://github.com/iTwin/viewer-components-react/assets/22688247/5bb7ab23-3e57-4380-93a5-e63a3e3d3af6

[Uyen]: The modal is updated with issue level icons and custom filter.
[custom filter.webm](https://github.com/iTwin/viewer-components-react/assets/56598021/868b588c-905f-413a-8268-967606636307)

### CODE REVIEW

File: packages/itwin/grouping-mapping-widget/src/widget/components/ExtractionStatusIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+export interface ExtractionStatusIconProps {
+  iconStatus: "negative" | "positive" | "warning";
+  onClick: () => void;
+  iconMessage: string;
+}
+
+export const ExtractionStatusIcon = ({ iconStatus, onClick, iconMessage }: ExtractionStatusIconProps) => {
+  return (
+    <>
```
Comment (Guillar1) [inline packages/itwin/grouping-mapping-widget/src/widget/components/ExtractionStatusIcon.tsx:17]: Fragment not needed.

File: packages/itwin/grouping-mapping-widget/src/widget/components/ExtractionMessageModal.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            Header: "Message",
+            accessor: "message",
+            width: "25vw",
+          },
+        ],
+      }], []
+  );
+
+  return (
+    <>
```
Comment (Guillar1) [inline packages/itwin/grouping-mapping-widget/src/widget/components/ExtractionMessageModal.tsx:49]: Fragment not needed.

File: packages/itwin/grouping-mapping-widget/src/widget/components/ExtractionStatusIcon.tsx (hunk 1 above)
Comment (smmr-dn) [reply packages/itwin/grouping-mapping-widget/src/widget/components/ExtractionStatusIcon.tsx]: I think I got it fixed

File: packages/itwin/grouping-mapping-widget/src/widget/components/ExtractionMessageModal.tsx (hunk 2 above)
Comment (smmr-dn) [reply packages/itwin/grouping-mapping-widget/src/widget/components/ExtractionMessageModal.tsx]: I also got this fixed.

**Comment by smmr-dn** [comment]
Mistyped username in git config so Git thought I was another user which triggered CLA.

**Comment by smmr-dn** [comment]
CLA problem

**Pull request closed without merging**